pwndbg/commands/hexdump.py

```diff
@@ -17,6 +17,13 @@
 pwndbg.config.Parameter('hexdump-bytes',
                          64,
                          'number of bytes printed by hexdump command')
+pwndbg.config.Parameter('hexdump-group-width',
+                         4,
+                         "number of bytes grouped in hexdump command. If -1, the architecture's pointer size is used.")
```

Comment:
```suggestion
                         "number of bytes grouped in hexdump command (If -1, the architecture's pointer size is used)")
```

This will make it more readable when the message from `set param val` is printed